SimCalorimetry/EcalSimAlgos/src/ComponentSimParameterMap.cc

```diff
@@ -0,0 +1,49 @@
+#include "SimCalorimetry/EcalSimAlgos/interface/ComponentSimParameterMap.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
+#include <iostream>
+
+/** Relevant constants are:
+  4.5 photoelectrons per MeV (J. Nash's slides)
```

Comment:
Which slides?

Reply:
I've removed that -- that was copypasted from EcalSimParameterMap.cc by mistake.